Modules/Core/Common/include/itkCellInterface.h

```diff
@@ -461,7 +461,7 @@ class ITK_TEMPLATE_EXPORT CellInterface
 
 #endif
 
-  /** Standard part of every itk Object. */
+  /** Defines the virtual member function `GetNameOfClass()`, which returns the name of this class as a string. */
```

Comment:
Looking at my own proposal again, the words "as a string" seem somewhat superfluous 🤔

Reply:
Does it return `std::string` or `char *`? 😄

Reply:
https://github.com/InsightSoftwareConsortium/ITK/blob/adcc46bf9600b78bda422c49a8aca4611c1751d1/Modules/Core/Common/include/itkMacro.h#L441-L442

So we could make it instead:
```cpp
  /** Defines `virtual const char * GetNameOfClass() const`, which returns the name of this class. */
```

Do you like that better?

(I would like it to fit on one line of code anyway.)

Reply:
And then of course, a similar line of documentation for itkOverrideGetNameOfClassMacro calls, like:

    /** Defines `const char * GetNameOfClass() const override`, which returns the name of this class. */

Right?